pkg/get/tools.go

```diff
@@ -415,5 +415,30 @@ https://releases.hashicorp.com/terraform/{{.Version}}/terraform_{{.Version}}_{{$
 https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/gh_{{.Version}}_{{$osStr}}_{{$archStr}}.{{$extStr}}`,
 		})
 
+	tools = append(tools,
```

Comment:
We will need a unit test for the app. Please can you see other examples like for faas-cli / inlets?